fbfmaproom/data-conversion-scripts/oromia/bad-years-v3-3season.py

```diff
@@ -0,0 +1,57 @@
+import cftime
+import datetime
+import pandas as pd
+import xarray as xr
+import sys
+import os
+
+
+dirout='/data/aaron/fbf-candidate/'
+regionsdic = {
+    "ethiopia/southern-oromia": {"file": "/data/aaron/fbf-candidate/original-data/oromia/bad_years.csv"},
+  "djibouti": {"file": "/data/aaron/fbf-candidate/original-data/djibouti/bad_years.csv"}
```

Comment:
Why is there only one file for Djibouti? The Trello card has three, one for each season.

Reply:
This is because the script is designed so that a single file contains all  seasons. Therefore, if there is an update or a new season, it only  is added (or updated) in one file. That's why I asked earlier if it makes sense that FbF group to be responsible for maintaining and sharing a single file with updates instead of N files 

Example: 
File:
year, jas, mam, ond
1991 , 6. , 17,  6
1992, 33, 10, 33
1993, 6,  33, 33

then:
python bad-years-v3-3season.py ond djibouti 
python bad-years-v3-3season.py jas djibouti.  
.......... etc

Reply:
I see. So you combined the three files into one by hand, right? I prefer to avoid manual steps, because it's hard to keep track of how they were done, and to identify and fix bugs when we make a mistake.

I agree that for the future, it's a good idea to define a standard file format and asking FIST to adhere to it is a good idea, so we don't have to keep writing one-off scripts to parse different formats.